### PR TITLE
doc: improve reply raw explaination

### PR DESCRIPTION
**Pull request by zekth**
fix https://github.com/fastify/fastify/issues/2372

Added some doc to explain the issue of using `Reply.raw`

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Reply.md [hunk 1]
```diff
@@ -391,3 +391,21 @@ For more details, see:
 - https://github.com/fastify/fastify/issues/1864 for the discussion about this feature
 - https://promisesaplus.com/ for the definition of thenables
 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then for the signature
+
+<a name="in addition"></a>
+### In addition
```
Comment (Eomm) [inline docs/Reply.md:396]: I would call it `.raw` since there is not a brief paragraph that explains it in detail

Comment (zekth) [reply docs/Reply.md:396]: Make sense, i'll expand the section

**Review by mcollina — APPROVED** [review]
lgtm

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Review by fox1t — APPROVED** [review]
LGTM

**Pull request merged**